### PR TITLE
Add example to Basic Usage in doc

### PR DESCRIPTION
**Pull request by densmirn**
image

### CODE REVIEW

**Comment by akharche** [comment]
@densmirn Maybe change description somehow to avoid words repetition?

The code below illustrates a typical ML workflow that consists of data pre-processing and predicting stages. Intel SDC is intended to compile data pre-processing stages that includes reading dataset from a csv file, data filtering and performing Pearson Correlation operation. The prediction based on   gradient boosting regression module is made using scikit-learn module.

**Comment by akharche** [comment]
@samaid Can we use your script based on "prices" dataset here as an example of SDC usage?

**Comment by Hardcode84** [comment]
Tests failed.

When I tried to run it locally on pure pandas (0.25.3) it is also failed. Had to change `columns=('symbol', 'volume')` -> `columns=['symbol', 'volume']`

**Comment by densmirn** [comment]
> Tests failed.
> 
> When I tried to run it locally on pure pandas (0.25.3) it is also failed. Had to change `columns=('symbol', 'volume')` -> `columns=['symbol', 'volume']`

The example shouldn't be ran on pure Pandas. We need `sklearn` to run the example. So these are 2 options. First one is to skip the example somehow. Second one is to install `sklearn` to CI environment. @Vyacheslav-Smirnov what option is better to select and how to reach selected option?

**Comment by densmirn** [comment]
> > Tests failed.
> > When I tried to run it locally on pure pandas (0.25.3) it is also failed. Had to change `columns=('symbol', 'volume')` -> `columns=['symbol', 'volume']`
> 
> The example shouldn't be ran on pure Pandas. We need `sklearn` to run the example. So these are 2 options. First one is to skip the example somehow. Second one is to install `sklearn` to CI environment. @Vyacheslav-Smirnov what option is better to select and how to reach selected option?

Exclude the example from common pipeline by skipping it.

**Comment by Vyacheslav-Smirnov** [comment]
@densmirn ,
I would prefer not to skip any example - they all should be executed and all should be passed.
To add sklearn (is it scikit-learn??) to CI for example build just add required package to https://github.com/IntelPython/sdc/pull/761/files#diff-12db1f3244c35f97542b6b38613320cfL82
sdc_utils.create_environment(['scipy'])

**Comment by Vyacheslav-Smirnov** [comment]
Also if this example requires additional package this package should be mentioned in the README.rst in sections related to examples.

File: buildscripts/run_examples.py [hunk 1]
```diff
@@ -56,7 +65,8 @@ def run_examples(sdc_utils):
             passed += 1
             sdc_utils.log_info(f'{sdc_example} PASSED')
 
-    sdc_utils.log_info(f'SDC examples summary: {total} RUN, {passed} PASSED, {failed} FAILED', separate=True)
+    summary_msg = f'SDC examples summary: {total} RUN, {passed} PASSED, {failed} FAILED, {skipped} SKIPPED'
+    sdc_utils.log_info(summary_msg, separate=True)
```
Comment (Vyacheslav-Smirnov) [inline buildscripts/run_examples.py]: @densmirn ,
I would prefer not to skip any example - they all should be executed and all should be passed.
To add sklearn (is it scikit-learn??) to CI for example build just add required package to https://github.com/IntelPython/sdc/pull/761/files#diff-12db1f3244c35f97542b6b38613320cfL82
sdc_utils.create_environment(['scipy'])

Comment (Vyacheslav-Smirnov) [reply buildscripts/run_examples.py]: Like sdc_utils.create_environment(['scipy', 'sklearn'])

Comment (densmirn) [reply buildscripts/run_examples.py]: Done!

Comment (densmirn) [reply buildscripts/run_examples.py]: I hope it won't slow down CI.

Comment (densmirn) [reply buildscripts/run_examples.py]: I skipped the example again, because the example uses big external dataset as input. So we are not able to run it in CI. This example is just for documentation and successfully works with downloaded dataset on the local system.

Comment (densmirn) [reply buildscripts/run_examples.py]: @Vyacheslav-Smirnov are you ok with that? If yes, please re-review the PR, if not, let's think up about another solution, but I don't think we have enough time before release.

Comment (Vyacheslav-Smirnov) [reply buildscripts/run_examples.py]: I just do not like the idea of skipping examples in CI) They all should work.
My thought is - can we just include the example in doc without adding it in examples?
Or maybe there is some less restrictive example?
If none of above suits - then lets merge this PR for now because the documentation need to be completed.

Comment (densmirn) [reply buildscripts/run_examples.py]: I would like to let user run the example directly from repo, so the file should be located there. Maybe we can think up about less restrictive example in the future. Now I prefer to merge the PR. Thanks!
